dist/node/meyda-wa.js

```diff
@@ -45,11 +45,13 @@ var MeydaAnalyzer = function () {
 		self.barkScale = utilities.createBarkScale(self.bufferSize, self.sampleRate, self.bufferSize);
 		self.melFilterBank = utilities.createMelFilterBank(self.melBands, self.sampleRate, self.bufferSize);
 
+		self.inputData = null;
+
 		self.spn.onaudioprocess = function (e) {
 			// self is to obtain the current frame pcm data
-			var inputData = e.inputBuffer.getChannelData(0);
+			self.inputData = e.inputBuffer.getChannelData(0);
```

Comment:
I'm sure I did this yesterday. Maybe it was part of my enabling buffer comparisons which hasn't been merged yet...